apps/modernization-api/src/main/java/gov/cdc/nbs/patient/search/SearchablePatientConverter.java

```diff
@@ -168,7 +168,10 @@ private static List<NestedEntityId> asIdentifications(final Person person) {
   }
 
   private static NestedEntityId asIdentification(final EntityId identification) {
-    String adjusted = identification.getRootExtensionTxt().replaceAll("\\W", "");
+    String value = identification.getRootExtensionTxt();
```

Comment:
nice.  could we add Optional to the schema across the board and reveal all the null ptr issues?